css/style.css

```diff
@@ -0,0 +1,141 @@
+:root {
+    --color-light: oldlace;
+    --color-dark: #333;
+    --color-primary: rgb(148, 226, 148);
+    --color-primary-dark: rgb(53, 73, 53);
+}
+ 
+ * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* Setting the value of 1rem */
+html {
+    /* 1rem = 10 pixels */
+    font-size: 62.5%;
+}
+ 
+body {
+    font-family: "Nanum Gothic", sans-serif;
+    font-size: 1.6rem;
+    background-color: var(--color-light);
+    color: var(--color-dark);
+}
+
+a {
+    text-decoration: none;
+    color: var(--color-dark);
+}
+
+ul {
+    list-style: none;
+}
+
+img {
+    display: block;
+    width: 100%;
+    max-width: 100%;
+}
```

Comment:
**Stylelint Error:** _Expected indentation of 2 spaces_
Please check indentation throughout the file, every line of attribute should have only 2 spaces of indentation instead of 4.